adhocracy4/maps/fields.py

```diff
@@ -0,0 +1,41 @@
+from django.core import validators as django_validators
+from django.utils.translation import ugettext_lazy as _
+from jsonfield.fields import JSONField, JSONFormField
+
+
+class GeoJSONFormField(JSONFormField):
+
+    def to_python(self, value):
+        empty_featureset = '{"type":"FeatureCollection","features":[]}'
+        if (value not in django_validators.EMPTY_VALUES and not
+                value == empty_featureset):
+            return super().to_python(value)
+        else:
+            return None
+
+
+class GeoJSONField(JSONField):
```

Comment:
You can overwrite this in the [form](https://github.com/liqd/a4-opin/blob/master/euth/maps/forms.py) if needed. Which according to the [django docs](https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/#overriding-the-default-fields) can happen even in nicer syntax.

If that is not suitable we can probably also change the error message here.